### ISSUE TITLE
Graph layout for mutliple entities

### ISSUE THREAD

**Issue by grtlr** [rerun-io/rerun#8126]
#7500 implements a basic graph viewer. One thing that I'm not entirely sure about is how to handle multiple entities in a single graph viewer.

Some context: Since graph layouts are hard (and users have different preferences) we wanted to be able to specify node positions for each node in the new `GraphNodes` archetype. We now have to think about how we want to show multiple entities in the graph viewer. In particular, I'm wondering if the entities should:

1. all share the same global coordinate system and a global layouting algorithm
2. each entity has its own local coordinate system within a common viewer each with its own layouting algorithm

So far @abey79 and I have leaned towards (2) to prevent overlap between graphs from different entities. Thinking some more about this, we probably should go with approach (1), as the overlap can be a feature: it will allow the user to compose larger graphs from smaller entities (each with their own properties).

The biggest benefit for (2) is that we can perform overlap removal for multiple entities. Arguably, in that case the user wants the visualize multiple graphs at once that don't share a common meaning. For this, having multiple separate graph viewers (each with their own query) side by side is likely a much better option.

To summarize: the big downside of automatic overlap removal (2) is that the user will ultimately loose control over the exact node positions for graphs composed of multiple entities. Because of this, I think we should do (1), with the downside of potentially having some overlaps + a more complex global layout algorithm.

**Comment by abey79** [comment]
done
